dblookupext/historyRepository.go

```diff
@@ -122,14 +122,7 @@ func NewHistoryRepository(arguments HistoryRepositoryArguments) (*historyReposit
 
 // RecordBlock records a block
 // This function is not called on a goroutine, but synchronously instead, right after committing a block
-func (hr *historyRepository) RecordBlock(
-	blockHeaderHash []byte,
-	blockHeader data.HeaderHandler,
-	blockBody data.BodyHandler,
-	scrResultsFromPool map[string]data.TransactionHandler,
-	receiptsFromPool map[string]data.TransactionHandler,
-	logs map[string]data.LogHandler,
-) error {
+func (hr *historyRepository) RecordBlock(blockHeaderHash []byte, blockHeader data.HeaderHandler, blockBody data.BodyHandler, scrResultsFromPool map[string]data.TransactionHandler, receiptsFromPool map[string]data.TransactionHandler, logs []*data.LogData, ) error {
```

Comment:
the additional `,` at the end, i think `gofmt` will remove that

Reply:
done, thanks